docs/using.rst

```diff
@@ -181,6 +181,19 @@ To use ADB with Host-Based tools
 .. _Docker Tooling: http://www.eclipse.org/community/eclipse_newsletter/2015/june/article3.php
       **Note:** Testing has been done with Eclipse 4.5.0.
 
+Using the ADB behind http proxy
+===============================
+
+Today, most of the organizations enable proxy servers for internal networks, the good news is, ADB can be set up behind this proxy. All you need to do is, un-comment the proxy variable in the Vagrantfile and assign the required value. 
+
```

Comment:
"Today, most of the organizations enable proxy servers for internal networks, the good news is," -> I think we should avoid the term "Today" as it kind of refers to a certain time line. But what we are trying to communicate is a generic thing and the rest of the sentence is not necessary as this seems like extra information wrt to how many organizations are using proxy. Also the witting style should be nearly in sync with other documentation above and below it.